virt-share/mount_share.te

```diff
@@ -0,0 +1,17 @@
+
+type mount_share, domain, data_between_core_and_vendor_violators;
+type mount_share_exec, exec_type, file_type, vendor_file_type;
+init_daemon_domain(mount_share)
+
+allow mount_share linkerconfig_file:dir getattr;
+allow mount_share metadata_file:dir getattr;
+allow mount_share mnt_user_file:dir { getattr search };
+allow mount_share overlayfs_file:dir getattr;
+allow mount_share system_data_file:dir getattr;
+allow mount_share vendor_toolbox_exec:file execute_no_trans;
+allow mount_share mnt_vendor_file:dir { search getattr };
+allow mount_share mnt_pass_through_file:dir search;
+allow mount_share user_profile_data_file:dir getattr;
+
+# allow mount_share mirror_data_file:dir { getattr search };
+permissive mount_share;
```

Comment:
Please remove permissive sentence in your final patch.

Reply:
Not able to remove permissive since mirror_data_file:dir cannot pass the build. I think we can add permissive here for this.

Reply:
User build doesn't allow the existence of permissive domains. With this line, user build will meet compilation error.